salesforce/Ballerina.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-distribution = "2201.2.0"
+distribution = "2201.2.1"
 org = "ballerinax"
 name = "salesforce"
 version = "7.1.0"
```

Comment:
Let's change the version to `7.2.0` @kasthuriraajan

Reply:
Ack. Thanks. I have updated